__init__.py

```diff
@@ -21,6 +21,10 @@
  ***************************************************************************/
```

Comment:
Please see https://github.com/inasafe/inasafe/wiki/Coding-Standards#standard-headers for the expected format of headers. In this case copyright would be to cadasta, not to InaSAFE....